### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by PicoCentauri**
* Added banners for pypi, conda, and the documentation
* Update installation instructions to inlude conda

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,14 +2,27 @@
 
 [![Test](https://github.com/lab-cosmo/scikit-matter/workflows/Test/badge.svg)](https://github.com/lab-cosmo/scikit-matter/actions?query=workflow%3ATest)
 [![codecov](https://codecov.io/gh/lab-cosmo/scikit-matter/branch/main/graph/badge.svg?token=UZJPJG34SM)](https://codecov.io/gh/lab-cosmo/scikit-matter/)
+[![PythonVersion](https://img.shields.io/badge/python-3.8%20%7C%203.9%20%7C%203.10%20%7C%203.11-orange)](https://pypi.org/project/maicos)
+[![pypi](https://img.shields.io/pypi/v/skmatter.svg)](https://pypi.org/project/skmatter)
+[![conda](https://anaconda.org/conda-forge/skmatter/badges/version.svg)](https://anaconda.org/conda-forge/skmatter)
+[![black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
```
Comment (Luthaf) [inline README.md:8]: Do we really need these banners? I don't see what they bring, especially the PythonVersion and black ones (as in I never see myself clicking on any of them)

Comment (PicoCentauri) [reply README.md:8]: What do the maintainers say @agoscinski and @rosecers ?

Comment (rosecers) [reply README.md:8]: I honestly couldn't care less... Happy to go with the group on this one.

Comment (PicoCentauri) [reply README.md]: I removed the python versions and the black banner. I think the other three are useful.

**Review by Luthaf — APPROVED** [review]
Works for me!

**Pull request merged**